docs/index.md

```diff
@@ -113,14 +113,19 @@ make -j 4
 You will need to source `env_standalone.sh` each time you want to use the package, or add it to your login.
 
 ### Standalone compilation with LCG
-LCG102 contains ROOT 6.26, which at the time of writing is not available in any CMSSW release.
-This repo can be compiled against LCG102 with:
-```
+LCG102 and above contains ROOT 6.26, which at the time of writing is not available in any CMSSW release.
```

Comment:
Very minor, I wonder if we might want to just change this line, to help future proof it from becoming out-of-date or misleading.

maybe something like:

```
For compilation outside of CMSSW, for example to use ROOT versions not yet available in CMSSW, one can compile against LCG releases. The current default is to compile with LCG_102 which contains ROOT 6.26.
```

Reply:
Thanks, that's a good suggestion